src/Aspire.Hosting.Azure.AppService/AzureAppServiceWebsiteContext.cs

```diff
@@ -362,6 +371,15 @@ private ProvisioningParameter AllocateParameter(IManifestExpressionProvider para
     {
         return parameter.AsProvisioningParameter(Infra, isSecure: secretType == SecretType.Normal);
     }
+    private void AddDashboardSettings(WebSite webSite, ProvisioningParameter acrClientIdParameter, ProvisioningParameter dashboardUri)
+    {
+        webSite.SiteConfig.AppSettings.Add(new AppServiceNameValuePair { Name = "OTEL_SERVICE_NAME", Value = resource.Name });
+        webSite.SiteConfig.AppSettings.Add(new AppServiceNameValuePair { Name = "OTEL_EXPORTER_OTLP_PROTOCOL", Value = "grpc" });
+        webSite.SiteConfig.AppSettings.Add(new AppServiceNameValuePair { Name = "OTEL_EXPORTER_OTLP_ENDPOINT", Value = "http://localhost:6001" });
```

Comment:
is `localhost:6001` correct here?

Reply:
It’s a sidecar within the same network as the main application